frontend/src/components/GymNavBar.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react'
```

Comment:
can remove React

Reply:
> src/components/GymNavBar.tsx
  Line 31:5:   'React' must be in scope when using JSX  react/react-in-jsx-scope
  Line 32:7:   'React' must be in scope when using JSX  react/react-in-jsx-scope
  Line 33:9:   'React' must be in scope when using JSX  react/react-in-jsx-scope
  Line 42:13:  'React' must be in scope when using JSX  react/react-in-jsx-scope
  Line 43:15:  'React' must be in scope when using JSX  react/react-in-jsx-scope

It caused some compiler error, any experience with this? Else I will keep it like this first

Reply:
O ya i think rhapp is not updated to latest so we still have to keep it in